corehq/apps/locations/util.py

```diff
@@ -298,7 +298,14 @@ def dump_locations(domain, download_id, include_consumption, headers_only, task=
     with open(path, 'rb') as file_:
         db = get_blob_db()
         expiry_mins = 60
-        db.put(file_, download_id, timeout=expiry_mins)
+        db.put(
+            file_,
+            domain=domain,
+            parent_id="BlobDownload",
```

Comment:
`LocationDownload`? Just thinking that if we're going to put something here it might as well be a bit more specific. Could also leave it blank if that's an option